integration/auth/auth_test.go

```diff
@@ -53,12 +61,379 @@ func TestMain(m *testing.M) {
 	if err != nil {
 		log.Fatalln(err)
 	}
-
 	os.Exit(m.Run())
 }
 
+func TestUserManagement(t *testing.T) {
+	t.Run("clean the slate", cleanupUsers)
```

Comment:
This will delete all users in the project, which is not a good idea. Look at the integration tests of other SDKs to see how they handle this.